docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/postgres.md

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_position: 1
+---
+
+# Postgres Destination
+
+In order to create an end-to-end test of the ELT pipeline apart from the Source which is the KYVE network, you should
+configure a destination where the `records` will be stored.
+
+This page will guide you through setting up an ELT pipeline to fetch data from
+a [KYVE data pool](https://docs.kyve.network/basics/pools.html) and import it into a local `Postgres` database.
+
+## Postgres Local Deployment
```

Comment:
to keep in parity with the Snowflake docs, the `##` header would be:
```suggestion
## Postgres Setup
```

Reply:
LGTM.